CMSgov/design-system
# [WNMGDS-1402] Remove unnecessary nav els

## Summary

[Jira ticket](https://jira.cms.gov/browse/WNMGDS-1402)

For vertical nav: Added conditional logic to wrap link list with a `<nav>` only if it's not nested (using already existing `nested` prop).

For header: WIP

## How to test

For vertical nav: Run `yarn storybook` locally, navigate to http://localhost:6006/?path=/story/components-vertical-nav--default-vertical-nav and open developer tools. You should only find one instance of `<nav>` being used.

For header: Run `yarn storybook:healthcare` and navigate to Header component. Note only one `<nav>` in use.
